samples/sample_calibrate_eye_to_hand.py

```diff
@@ -75,6 +77,14 @@ def _main():
     else:
         print("FAILED")
 
+    print("Transforming calibration result")
```

Comment:
You are not "transforming the calibration result", but instead getting the calibration results transform. So I would update the print to reflect this.

Reply:
Updated it to say `"Getting calibration result transformation matrix"`